src/test/resources/application-integration-test.yml

```diff
@@ -1,3 +1,4 @@
+spring:
 cloud:
```

Comment:
in yaml the lines below the colon should be indented for it to be seen as `spring.cloud.<blah>`